### ISSUE TITLE
Create a Contributing.md file and add list of Project needs

### ISSUE THREAD

**Issue by catheraaine** [catheraaine/risk-legacy#2]
- Guts? File structure? Framework ??? 
- Files I'm missing, like `.gitignore`
- I have no build process
- I have no idea what I'm doing

**Comment by catheraaine** [comment]
![I have no idea what I'm doing](http://myfunnymemes.com/wp-content/uploads/2015/04/I-Have-No-Idea-What-Im-Doing-Star-Trek-Guy-Was-Actually-Twisting-a-See-Through-Tool-.gif)

**Comment by owenvoke** [comment]
@catheraaine would you like any help with this? 😄 I'm quite interested.

P.S. Is this looking to be a PHP sorta thing in future?

And, would it store games to a database? Or, would it need to store the data to a file (e.g. `savegame_-date-.risksave`) so you could easily send your saves to another player?

I was just thinking that this would be a fun project for me to help with 👍

**Comment by catheraaine** [comment]
Please do! 

I haven't chosen a backend language, thought mostly about node/json/js. I was thinking database, but I have no idea. It could write locally.

**Comment by owenvoke** [comment]
I guess I may not be able to help much :tongue. I hate node, but I'll have a look and maybe fork it to test some things out.

**Comment by beforan** [comment]
@catheraaine do you have any more thoughts on technologies for this? I think many of the other issues would be done differently depending on this decision.

You could definitely do this using html5 local storage and therefore make it really only frontendy client side javascript. In which case i would advocate picking up a framework like knockout or angular to give you decent data structures and templating.

However, if you did want to use node or something server side, and persist to a database of some kind (sql or no), then the frontend stuff becomes less useful as your data structures are server side and your templating technology is different (e.g. Jade for node/express).

I think a lot of the current stuff would benefit from templating (e.g. have a defined countries.json that everywhere uses, instead of repeatedly hard-coded the same countries into drop downs in HTML ;)) but I'm loathed to do it until I know whether to do it client side or server side?

Sorry for the essay.

TL;DR any direction on node, or frontend JS, or something else?
